frontend/src/components/Forms/Fields/index.tsx

```diff
@@ -84,16 +84,23 @@ export const FormInput = ({
               as={DatePicker}
               id={fieldName}
               label={fieldLabel}
-              inputFormat={DATE_FORMAT_SHORT_YEAR}
-              value={dayjs(fieldValue)}
-              fullWidth={true}
+              format={DATE_FORMAT_SHORT_YEAR}
+              value={null === fieldValue ? null : dayjs(fieldValue)}
```

Comment:
oh, dayjs returns a date if it gets passed an undefined

no wonder it's tricky.

Reply:
yea...